src/contexts/CalculatorsContext.tsx

```
@@ -0,0 +1,580 @@
+        },
+      };
+    case "CLEAR_HISTORY":
+      return {
+        ...state,
+        history: {
+          ...state.history,
+          [action.payload]: [],
+        },
+      };
+    case "RESET_CALCULATOR":
+      const resetKey = action.payload as keyof CalculatorState;
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)